### ISSUE TITLE
Multiselect dropdown FastPass errors: elements with an Aria role that require children contain them, elements with an Aria role that require parent roles are contained by them

### ISSUE THREAD

**Issue by caratyler** [microsoft/fluentui#20280]
- **Package version(s)**: 8.37.0 and 7.177.2
- **Browser and OS versions**: (fill this out if relevant)
- Note - I see several other dropdown accessibility issues but not this exact one -- e.g, 19518

### Describe the issue:
1. Activate the example for
v7: Multi-select uncontrolled example here:
https://fabricweb.z5.web.core.windows.net/oufr/7.177.2/#/examples/dropdown
v8: Multi-select uncontrolled example here:
https://fabricweb.z5.web.core.windows.net/oufr/8.37.0/#/examples/dropdown
2. Run Fast Pass

#### Actual behavior:

Accessibility insights is failing with the following errors:
- Certain ARIA roles must contain particular children
- Certain ARIA roles must be contained by particular parents>

#### Expected behavior:

Fast pass passes

### If applicable, please provide a codepen repro:

v7: Multi-select uncontrolled/controlled example here:
https://fabricweb.z5.web.core.windows.net/oufr/7.177.2/#/examples/dropdown
v8: Multi-select uncontrolled/controlled example here:
https://fabricweb.z5.web.core.windows.net/oufr/8.37.0/#/examples/dropdown

### Priorities and help requested (not applicable if asking question):
We are on v7 of Fluent so we are hoping to see a fix there, not just in v8

Are you willing to submit a PR to fix? (Yes, No)
Yes, or help cherry pick the fix back to v7 if necessary

Requested priority: (Blocking, High, Normal, Low)
High - Blocking grade C which is required for release in our organization

Products/sites affected: (if applicable)

**Comment by gouttierre** [comment]
@caratyler - Thanks for filing this issue with us. To set expectations the developers will review this issue once capacity allows. 

@khmakoto  - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley - I am looping you in as you are working with Dropdown convergence.

**Comment by smhigley** [comment]
This is an error with the automated testing ruleset, and fixed in axe-core v4.3.4: https://github.com/dequelabs/axe-core/releases/tag/v4.3.4

Accessibility Insights is still on 4.3.2.

**Comment by smhigley** [comment]
(the issue was with recognizing `group` as a valid child of `listbox`)
